CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Unreleased
 
 **Added**
```

Comment:
These are all great examples of atomic commits. I would merely recommend moving the relevant path into its respective commit.

![image](https://cloud.githubusercontent.com/assets/48100/24780629/adadb968-1aed-11e7-8402-99e2e3b23f88.png)